tools/panaroo/panaroo.xml

```diff
@@ -0,0 +1,156 @@
+            #end if
+            #if $aligner:
+                --aligner '$gene_alignment.aligner'
+            #end if
+    ]]></command>
+    
+    <inputs>
+        <param type="data" name="input1" format="gff,gff3" multiple="true" label="Select gff inputs to Panaroo" help="Select the files you wish to analyze with Panaroo, must be in gff3 format."/>
+        <!--MODES-->
+        <section name="modes" title="Modes"  expanded="false">
+            <param argument="--clean-mode" type="select" label="filtering mode" help="Change a number of defaults to either favour strict, moderate or sensitive filtering">
+                <option value="strict" selected="True">Strict</option>
+                <option value="moderate">Moderate</option>
+                <option value="sensitive ">Sensitive</option>
+            </param>
+            <param name="remove_invalid_genes" type="boolean" optional="true" truevalue="--remove-invalid-genes" falsevalue="" label="Remove Invalid Genes" help="Removes annotations that do not conform to the expected Prokka format. Disabled by default."/>
+        </section>
+        <!--Matching-->
+        <section name="matching" title="Matching"  expanded="false">
+            <param argument="--threshold" type="float" optional="true" value="0.98" label="Initial Clustering Threshold" help="Set the threshold for gene clustering. Default is 0.98."/>
+            <param argument="--family_threshold" type="float" optional="true" value="0.7" label="Family Sequence Identity Level" help="Set the level at which Panaroo collapses genes into putative families. Default is 0.7."/>
+            <param argument="--len_dif_percent" type="float" optional="true" value="0.98" label="Length Difference Percent" help="Set the length difference percent for initial gene clustering. Default is 0.98."/>
+            <param name="merge_paralogs" type="boolean" optional="true" truevalue="--merge_paralogs" falsevalue="" label="Merge Paralogs" help="Merging paralogs can be enabled. Disabled by default."/>
+        </section>
+        <!--Refind-->
+        <section name="refind" title="Refind"  expanded="false">
+            <param argument="--search_radius" type="integer" optional="true" value="5000" label="Refinding Search Radius" help="Set radius for the search in nucleotides for refinding genes. Default is 5000."/>
+            <param argument="--refind_prop_match" type="float" optional="true" value="0.5" label="Refinding Proportion Match" help="Set the proportion of the missing gene to consider as a match for refinding. Default is 0.5."/>
+            <param argument="--refind-mode" type="select" optional="true" label="Refind Mode" help="Set the stringency mode at which to re-find genes.">
+                <option value="default" selected="True">Default</option>
+                <option value="strict">Strict</option>
+                <option value="off ">Off</option>
+            </param>
+        </section>
+        <!--GRAPH CORRECTION -->
+
```

Comment:
Would be nice if you add parameters for graph corrections as well, as the panaroo modes can be only be configured in the graph correction section.